aptiblejs/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "aptible.js",
+  "version": "0.0.1",
+  "description": "aptible.com frontend library",
+  "main": "index.js",
+  "scripts": {
+    "dev": "webpack --mode development --watch",
+    "build": "webpack --mode production"
```

Comment:
Can you please run this as part of the build in `.travis.yml` (and either don't check in the resulting asset, or make sure it's up to date?)?

As-is, we might end up forgetting to rebuild the asset when making changes